doc/invites.md

```diff
@@ -0,0 +1,21 @@
+Guidelines for Instruqt invites
+===============================
+
+* Public title: `Replicated Hands-On Labs for [Vendor Name]`
+* Public description: `An opportunity for the [Vendor Name] team to explore the features and capabilities of Replicated in a lab environment.`
```

Comment:
Maybe add another comment that says something like "at a minimum…" so that it's clear we can put more in there when it makes sense.

Reply:
Will do in a follow-up PR